dex/src/main/scala/com/wavesplatform/dex/tool/package.scala

```diff
@@ -0,0 +1,20 @@
+package com.wavesplatform.dex
+
+package object tool {
+
+  object PrettyPrintHelper {
```

Comment:
Why not just in `dex/src/main/scala/com/wavesplatform/dex/tool/PrettyPrintHelper.scala` ?

Reply:
It's too small and not so important to define in its own file. But yes, maybe, it would be better to place in package helpers